### PR TITLE
Updates YouTubeiOSPlayerHelper to 1.0.4

### PR DESCRIPTION
**Pull request by sasha-borodin**
Hey Chuck!  This PR includes a newly compiled and assembled `.xcframework`, from the latest release of the bound library (v1.0.4).  Some of the NuGet mechanics have also changed in order to work correctly with the `.xcframework`.  Finally, I updated the docs.

### CODE REVIEW

File: YouTubeiOSPlayerHelper.xcframework/ios-x86_64-simulator/YouTubeiOSPlayerHelper.framework/Headers/YTPlayerView.h [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+/**
+ * Callback invoked when an error has occured.
+ *
+ * @param playerView The YTPlayerView instance where the error has occurred.
+ * @param error YTPlayerError containing the error state.
+ */
+- (void)playerView:(nonnull YTPlayerView *)playerView receivedError:(YTPlayerError)error;
+
+/**
+ * Callback invoked frequently when playBack is plaing.
```
Comment (chotaling1) [inline YouTubeiOSPlayerHelper.xcframework/ios-x86_64-simulator/YouTubeiOSPlayerHelper.framework/Headers/YTPlayerView.h:108]: nit: typo on "plaing" 🤣

**Review by chotaling1 — COMMENTED** [review]
Are you depending on the nuget package for this, or do you already have it bound locally? I can try and get it updated some time today, if you are depending on it.

File: YouTubeiOSPlayerHelper.xcframework/ios-x86_64-simulator/YouTubeiOSPlayerHelper.framework/Headers/YTPlayerView.h (hunk 1 above)
Comment (sasha-borodin) [reply YouTubeiOSPlayerHelper.xcframework/ios-x86_64-simulator/YouTubeiOSPlayerHelper.framework/Headers/YTPlayerView.h:108]: I deserve this as payback for all my nits 😂

**Comment by sasha-borodin** [comment]
> Are you depending on the nuget package for this, or do you already have it bound locally? I can try and get it updated some time today, if you are depending on it.

We have our own internal NuGet repo that's been updated with a package with my changes. But if anyone else references your NuGet, they will benefit from the PR.

**Comment by chotaling1** [comment]
@sasha-borodin - PR has been merged, uploaded, and is awaiting verification. Thanks again for doing this!

**Pull request merged**